py_gtfs_rt_ingestion/ingest.py

```diff
@@ -83,26 +83,52 @@ def lambda_handler(event: dict, context) -> None:
     batch files should all be of same ConfigType
     """
     logging.info("Processing event:\n%s" % json.dumps(event, indent=2))
-    temp_dir = None
-    if 'files' not in event:
-        temp_dir = tempfile.TemporaryDirectory()
-        files = [download_file_from_s3(event['bucket'], file, temp_dir.name)
-                 for file in event['s3_files']]
```

Comment:
not downloading anymore. instead we'll read directly inside of the `s3_to_pyarrow` function. this is all contained inside of the thread.